module/Makefile.bsd

```diff
@@ -25,6 +25,7 @@ CFLAGS+= -I${INCDIR}/os/freebsd
 CFLAGS+= -I${INCDIR}/os/freebsd/spl
 CFLAGS+= -I${INCDIR}/os/freebsd/zfs
 CFLAGS+= -include ${INCDIR}/os/freebsd/spl/sys/ccompile.h
+CFLAGS+= -I${INCDIR}/..
```

Comment:
Would that be cleaner as -I${SRCDIR}?

Reply:
Actually after my PR https://github.com/openzfs/zfs/pull/10493 you'd want -I${.OBJDIR} instead.

Reply:
Also, on linux we actually pull it into every compilation via `-include`. Should freebsd do the same?

Another thing here is that ${INCDIR}/spl no longer exists, should be removed from CFLAGS.